api-specs/openrpc-dapp-remote-api.json

```diff
@@ -0,0 +1,480 @@
+{
+    "openrpc": "1.2.6",
+    "info": {
+        "title": "Splice Wallet JSON-RPC Remote dApp API",
+        "version": "0.1.0",
+        "description": "An OpenRPC specification for the dapp to interact with a wallet kernel."
+    },
+    "methods": [
+        {
+            "name": "status",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "title": "StatusResult",
+                    "type": "object",
+                    "properties": {
+                        "kernel": {
+                            "$ref": "#/components/schemas/KernelInfo"
+                        },
+                        "isConnected": {
+                            "title": "isConnected",
+                            "type": "boolean",
+                            "description": "Whether or not a connection to a network is established."
+                        },
+                        "chainId": {
+                            "title": "chainId",
+                            "type": "string",
+                            "description": "A CAIP-2 compliant chain ID, e.g. 'canton:da-mainnet'."
+                        }
+                    },
+                    "required": ["kernel", "isConnected"]
+                }
+            },
+            "description": "Returns the current chainId if connected."
+        },
+        {
+            "name": "connect",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "title": "ConnectResult",
+                    "type": "object",
+                    "properties": {
+                        "kernel": {
+                            "$ref": "#/components/schemas/KernelInfo"
+                        },
+                        "isConnected": {
+                            "title": "isConnected",
+                            "type": "boolean",
+                            "description": "Whether or not a connection to a network is esablished."
+                        },
+                        "chainId": {
+                            "title": "chainId",
+                            "type": "string",
+                            "description": "A CAIP-2 compliant chain ID, e.g. 'canton:da-mainnet'."
+                        },
+                        "userUrl": {
+                            "$ref": "#/components/schemas/UserUrl"
+                        }
+                    },
+                    "required": ["kernel", "isConnected", "userUrl"]
+                }
+            },
+            "description": "Ensures ledger connectivity. Returns the current chainId if connected, or the user url if disconnected."
+        },
+        {
+            "name": "darsAvailable",
```

Comment:
I noticed this spec contains a copy of every method from the general spec. I think we should omit these for clarity, showing only the methods that require special attention. 

This means that the Splice provider would need to use both rpc clients in tandem, but I think this is okay.